exampleSite/content/about-split/index.md

```diff
@@ -4,9 +4,12 @@ lastmod = "2017-10-05"
 date = "2017-10-05"
 subtitle = ""
 layout = "split"
-sidebar_title = "Sidebar"
-sidebar = """
-So much information   
+
+[[sidebar]]
+    title = "Sidebar"
+    sidebar = """
```

Comment:
Would probably call it content to be more consistent with the usual content parts.